content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,230 @@ section: doc
-* Choose a plan **standard** or **free**, the free plan only has one worker node and no subnet, to provision a standard cluster, you will need to upgrade your account to Pay-As-You-Go
-** To upgrade to a Pay-As-You-Go account, complete the following steps:
-. In the console, go to Manage > Account
-. Select Account settings, and click **Add credit card**.
-. Enter your payment information, click **Next**, and submit your information
-* Choose **classic** or **VPC**, read the https://cloud.ibm.com/docs/containers?topic=containers-infrastructure_providers[docs] and choose the most suitable type for yourself
-
-image::tutorials/IBMCloud/infra-select.png[]
-
-* Now choose your location settings, for more information please visit https://cloud.ibm.com/docs/containers?topic=containers-regions-and-zones#zones[Locations]
-** Choose **Geography** (continent)
-
-image::tutorials/IBMCloud/location-geo.png[]
-
-* Choose **Single** or **Multizone**, in single zone your data is only kept in on datacenter, on the other hand with Multizone it is distributed to multiple zones, thus  safer in an unforeseen zone failure
-
-image::tutorials/IBMCloud/location-avail.png[]
-
-* Choose a **Worker Zone** if using Single zones or **Metro** if Multizone
-
-image::tutorials/IBMCloud/location-worker.png[]
-
-* If you wish to use Multizone please set up your account with https://cloud.ibm.com/docs/dl?topic=dl-overview-of-virtual-routing-and-forwarding-vrf-on-ibm-cloud[VRF] or https://cloud.ibm.com/docs/vlans?topic=vlans-vlan-spanning#vlan-spanning[enable Vlan spanning]
-** If at your current location selection, there is no available Virtual LAN, a new Vlan will be created for you
-
-* Choose a **Worker node setup** or use the preselected one, set **Worker node amount per zone**
-
-image::tutorials/IBMCloud/worker-pool.png[]
-
-* Choose **Master Service Endpoint**,  In VRF-enabled accounts, you can choose private-only to make your controller accessible on the private network or via VPN tunnel. Choose public-only to make your controléer publicly accessible. When you have a VRF-enabled account, your cluster is set up by default to use both private and public endpoints. For more information visit https://cloud.ibm.com/docs/account?topic=account-service-endpoints-overview[endpoints].
-
-image::tutorials/IBMCloud/endpoints.png[]
-
-* Give cluster a **name**
-
-image::tutorials/IBMCloud/name-new.png[]
+To deploy Jenkins on IBM Cloud:
+
+. <<Provisioning a Kubernetes cluster,Provision a Kubernetes Cluster>>.
+If you already have a Kubernetes cluster, you can skip to step 2.
+. <<Deploying the IBM Cloud Block Storage plug-in,Deploy IBM Cloud Block Storage plug-in>>.
+If you have already deployed the the IBM Cloud Block Storage plug-in, you can skip to step 3.
+. <<Deploying Jenkins,Deploy Jenkins>>.
+. <<Verifying the Jenkins installation,Verify the jenkins installation>>.
+
+== Provisioning a Kubernetes cluster
+
```

Comment:
Generally, I recommend including a small intro for each section, even if only a sentence.

It can seem a little redundant, but I also recommend including a brief lead up to the steps. 

```suggestion
When deploying Jenkins to IBM Cloud, you must first provision a Kubernetes cluster. 

NOTE: If you already have a Kubernetes cluster, you can skip to <<Deploying the IBM Cloud Block Storage plug-in>>.

To provision a Kubernetes cluster:
```